src/dlgConnectionProfiles.cpp

```diff
@@ -118,17 +119,7 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
         cursor.clearSelection();
```

Comment:
The assertion on line 104 up above is getting triggered for me... the code is kind of complicated to figure out what is wrong at a glance. @SlySven any ideas?

Reply:
Yeah, see comment above - need a space after the `%1`...

Reply:
Done.